ocf-servers/start-ocf-servers-in-docker.sh

```diff
@@ -1,17 +1,75 @@
 #!/bin/bash
 
+# project home directory
+PROJ_HOME=/opt/SmartHome-Demo
+
+# direcotry for ocf servers
+OCF_DIR=$PROJ_HOME/ocf-servers/js-servers
+
+# configuration file name
+CONFIG_FILE=$PROJ_HOME/ocf-servers/ocf-server.conf
+
 # Print out container IP address
 echo "This container IP address is: `hostname -i`"
 
 # Set-up the path to where the node.js modules were installed
-export NODE_PATH=/opt/SmartHome-Demo/ocf-servers/node_modules/
+export NODE_PATH=$PORJ_HOME/node_modules/
```

Comment:
Good catch! Qiu is right. It was not found cos they are in the same directory. Thanks for correcting it for me. @gvancuts